src/firefly/java/edu/caltech/ipac/firefly/server/query/SearchServerCommands.java

```diff
@@ -79,6 +96,63 @@ public String doCommand(SrvParam params) throws Exception {
             }
         }
     }
+
+    public static class TableSearchSpatialBinary extends ServerCommandAccess.HttpCommand{
+
+        public void processRequest(HttpServletRequest req, HttpServletResponse res, SrvParam params) throws Exception {
+            TableServerRequest tsr = params.getTableServerRequest();
+            var inclColStr= tsr.getInclColumns();
+            var cAry= Arrays
+                    .stream(inclColStr.split(","))
+                    .map( (s) -> s.charAt(0)=='"' && s.charAt(s.length()-1)=='"' ? s.substring(1, s.length()-1) : s )
+                    .toList();
+
+            if (cAry.size()<2) return;
```

Comment:
Here, you extract ra/dec from the includeColumns field with assumption that ra is in index 0, then dec at 1 later down the road, accounting for quoted syntax used by SQL.
You could have made 'ra' / 'dec' parameters, then setInclColumns with those values.  This make the request more clear, making it easier to validate as well.